lib/fluent/plugin/buffer/file_chunk.rb

```diff
@@ -74,7 +74,7 @@ def append(data)
         end
 
         def concat(bulk, bulk_size)
```

Comment:
Off topic. Argument name should follow base class. `Chunk#concat` signature is `def concat(bulk, records)`.